hikari/impl/bot.py

```diff
@@ -448,15 +483,16 @@ def run(
         *,
         activity: typing.Optional[presences.Activity] = None,
         afk: bool = False,
+        idle_since: typing.Optional[datetime.datetime] = None,
```

Comment:
i kinda did alphabetise these so it was easier to scan a list.

We should decide on how to do this consistently as me and you keep doing the complete opposite lol.

I'm happy to discuss which way we should follow going forwards.

Reply:
I generally just bundle them up with the other ones that act the same. Eg: activity, afk, idle_since and status all control the bot status that gets displayed, so i put them next to each other. I can revert this if you want

Reply:
As long as it is consistent, it is fine. I just personally find alphabetised stuff easier to spot missing things in, etc.